pkg/consensus/recovery_message.go

```diff
@@ -0,0 +1,172 @@
+package consensus
+
+import (
+	"github.com/CityOfZion/neo-go/pkg/io"
+	"github.com/CityOfZion/neo-go/pkg/util"
+	"github.com/pkg/errors"
+)
+
+type (
+	recoveryMessage struct {
+		PreparationHash     *util.Uint256
+		PreparationPayloads map[uint16]*preparationCompact
+		CommitPayloads      map[uint16]*commitCompact
+		ChangeViewPayloads  map[uint16]*changeViewCompact
+		PrepareRequest      *prepareRequest
+	}
+
+	changeViewCompact struct {
+		ValidatorIndex     uint16
+		OriginalViewNumber byte
+		Timestamp          uint32
+		InvocationScript   []byte
+	}
+
+	commitCompact struct {
+		ViewNumber       byte
+		ValidatorIndex   uint16
+		Signature        []byte
+		InvocationScript []byte
+	}
+
+	preparationCompact struct {
+		ValidatorIndex   uint16
+		InvocationScript []byte
+	}
+)
+
+const uint256size = 32
+
+func (m *recoveryMessage) DecodeBinary(r *io.BinReader) {
+	l := r.ReadVarUint()
```

Comment:
didn't get why is that variable called "l"

Reply:
**l**en